### ISSUE TITLE
Proposal Regarding PVE

### ISSUE THREAD

**Issue by phucnd** [bunicorndefi/BIPs#6]
Dear BUNIans,

First of all, we would like to express our gratitude to all of our patient supporters until this moment. It is no easy decision to choose to stay with the project during this whole challenging period. Thank you for always having our back & encouraging us every day.

As specified in the [Buni Island - Towards the Bunicorn Metaverse](https://github.com/bunicorndefi/BIPs/issues/5), Bunicorn Game is moving from the basic click-to-play experience to a true simulation game genre that involves strategic and tactical challenges.

Here's our upcoming product development and timeline:

- Dec 10 2021 - PvP Challenge
- Dec 24 2021 - PvP Arena
- Dec 15 2021 - Metaverse Starter Launchpad
- Dec 25 2021 - Mobile App for DEX
- Dec 30 2021 - Puzzle Game Beta Test
- Jan 21 2022 - New NFT Marketplace
- Feb 8 2022 - Buni Island Testnet
- Feb 28 2022 - Buni Island Mainnet (web based)
- Mar 31 2022 - Puzzle Game Mainnet
- Q2 2022 - Buni Island Mobile App
- Q3 2022 - Bunicorn Gaming Metaverse

We are in the final stage of developing the new PvP mode for the current game and planning to release both [PvP Challenge and Arena](https://github.com/bunicorndefi/BIPs/issues/1) this December. More details will be revealed in the next articles. Our next big update is Buni Island, and it tentatively takes roughly 2 months to get the beta version out. Bunicorn launchpad and game together bring resonance effects for players since players owning in-game characters will have certain advantages while taking part in IDOs on Metavere Starter launchpad. The Bunicorn new NFT marketplace allows users to trade NFTs from INO projects on Metaverse Starter and other creative content creators/artists like [TuNa - the Top 100 Binance NFT](https://blog.buni.finance/tuna-the-top-100-binance-nft-creator-to-join-bunicorn-core-team-db940da365d2).

Therefore, huge efforts dedicated to these product developments take precedence, so that we need to re-manage our resources to align with new directions and ensure the project's longevity. As the main current activity in the Bunicorn game, the PvE mode is leading to an infinite amount of minted tokens. According to our observation and experience, the click-to-earn game's nature to be hard to scale in many aspects, especially with the escalated snowball effect of the token (that's why Buni Island is introduced). We gave a more detailed explanation [here](https://github.com/bunicorndefi/BIPs/issues/5). Thus, we'd like to open a voting poll to go forward with the following actions too:

- Close the PvE battle. This action will stop BUR from being infinitely minted and persist the value of income and NFTs so far. The current BUR supply will be used in Buni Island designed with new complex gameplay along with the sharing economy that is expected to provide more rooms for development and encourage long-term sustainability for Bunicorn Game. As a result, BUR will once again gain its value. 
- Along with the PvE closing, we will open 02 new Bunicorn NFT staking pools worth 250K BUNI each for trainers from level 11 and bunicorns with 3-4-5 stars on a monthly basis until the Buni Island is ready. You can stake your in-game characters (trainers and bunicorns) and get BUNI back. Once staked in this pool, your NFT will be burnt and not be able to claim back. In case the PvE closing proposal is disapproved, this staking pool will not exist. 

Moreover, regardless of the PvE closing result, we still open the [daily Raidboss event](https://blog.buni.finance/bunicorn-daily-raid-boss-bab2bd681afd), and soon PvP and Puzzle mode in December. They are temporarily our main activity in Bunicorn Game until the Buni Island is released. The Dungeon event will not be open anytime soon - [Our explanation](https://t.me/bunicornswap_en/134140). The reward BUNI pool of Raidboss has been crawled from the BUSD-BUNI pool on Bunicorn DEX. 

We are aware that a sharp change like this needs to be carefully discussed and consulted among the community. That is why we made this proposal and ask you, the true game owners, to decide where it will head next. If you think this is a possible solution to the situation, kindly upvote the proposal. If not, kindly vote down, and we will continue to keep the PvE Battle running until Buni Island is ready. 

The voting period will last for 3 days from 3 AM Nov 29, 2021 (UTC) to 3 AM Dec 2, 2021. The eligible address for voting is required to hold at least 1 Bunicorn or 1 trainer or 3000 BUNI tokens.

Kindly consider carefully before making your voting decision. Your rightful vote and opinion is now the key to opening an entirely new world for Bunicorn Game. Let us know your thoughts on [Discord](https://discord.gg/Jfa5eebPJs).

No matter what is the final result of this vote, we will respect the decision of the community whether to close PvE or not because you're the true owner of the game.

Thank you for reading,

Bunicorn Team.

**Comment by HackTheGame3** [comment]
You said "still open the daily Raidboss event, and soon PvP and Puzzle mode in December"

I just want to confirm whether puzzle mode is just a test or not.
The puzzle mode will be profitable or just for experience?

**Comment by persewall** [comment]
The updates you describe sound good, I agree that this is a logical evolutionary path.

**Comment by gammaJPG** [comment]
Hi Phuc, in my opinion, and with due respect, this proposal was poorly handled. The fact that you proposed it, even if it gets rejected, shows bunicorns intentions to players (investors). 

I understand that the minting of BUR needs to be stopped eventually, but there is a lot of people that only make revenue from it, specially new players. The pvp and bosses are not an option for them, while there's no info about the puzzle game. As of right now, if  pve gets erased to try to save the bur price and value, those players will not have a reason to keep their NFTs, and prices on the market will go close to zero on the marketplace. The burning pool will only let the higher ones retain some value. This doesn't only affect prices, but bunicon's reputation as a whole. If people are forced to liquidate their assets from a lack of income, it can be seen as buni giving up on these players. Some thoughts like these have already been stated on the discord channel. 

I believe there needs to be some sort of enticement/ compensation to give players a reason to keep their assets, given in the form of future rewards (for when buniland gameplay releases), which could be anything from exp or materials to just cosmetic items. This way, some of the players will be willing to take the risk of keeping their assets hoping that they will receive some exlusive items or have an advantage over players that join only after buniland is released. I think this is the least that can be done for those that trusted the game on it's current state and joined just recently. I understand that you are trying to focus all efforts on the game development, but deciding how to take this next step is crucial. Even if you can't figure balanced rewards right now, points could be given, so thay once rewards are figured out you make an exchange from points to these rewards.

In other words, even if rejected, this proposal **looks** like  bunicorn giving up on players with lower investments, which damages market prices and bunicorn's reputation. So, even if it gets rejected, a lot of players might quit, since this proposal added uncertanity about their only source of revenue.

**Comment by sang1225** [comment]
The last battle suspension and this time PVE suspension show that the current development team lacks understanding of the market system.
Now, this isn't a coincidence or the player's fault, it's the developer's fault.
In P2E, the importance of dealing with the market system is higher than that of game development.
The top priority of the development team right now is to hire experts in the economy and market system.
Without a manpower pool capable of building a sophisticated market system and responding in real time, if the operation is messed up as before, it will inevitably go bankrupt.

**Comment by aluminick** [comment]
I've already accepted the fact that I won't be able to get my ROI because of this sudden change.
This guy is panicking and don't know what to do. Just spewing random ideas without any data to back it up if it's going to work or not. You're just doing TRIAL AND ERROR method with people's money SMH.
Are your decisions data-driven or because of the downtrend?

**Comment by aluminick** [comment]
At this point, this guy is really desperate in saving himself. Giving people no choice but to burn their crypto assets.

**Comment by Devil89Maycry** [comment]
> At this point, this guy is really desperate in saving himself. Giving people no choice but to burn their crypto assets.

did you forget that this is also a game, a competitive one. They are not forcing you to burn your token man. You are just waiting to earn money from us gamers. We should be the one complaining, not you. We are the one's who are burning it and enjoying it while earning back what we have burned. Did we complain? no!

**Comment by Devil89Maycry** [comment]
> I've already accepted the fact that I won't be able to get my ROI because of this sudden change. This guy is panicking and don't know what to do. Just spewing random ideas without any data to back it up if it's going to work or not. You're just doing TRIAL AND ERROR method with people's money SMH. Are your decisions data-driven or because of the downtrend?

IF you just played the game and didn't HODL all your BUR, then these doesn't happen, stop blaming a single person. You are just too selfish to burn a portion of your BUR that's  why things like these exist. All you know is blame someone. As if you are a perfect person. Show us what you have invested, I bet you just invested 1dollar.

**Comment by damijankante** [comment]
Guys, I used to believe that you have a good plan but sorry, no longer. As already said above, you don't really know how the market works and just making an update after another which is further destroying the game as it doesn't really fix the main issue. In the meantime, it is also destroying all the small and even medium players investments for which you clearly don't have any respect for. The game is sadly destined for bankruptcy.

The only thing that can save this game is hiring or consult with someone that runs a successful P2E game and knows how such a market works. I am not that expert, but am sure I can already recognize what won't work.

**Comment by Devil89Maycry** [comment]
> Guys, I used to believe that you have a good plan but sorry, no longer. As already said above, you don't really know how the market works and just making an update after another which is further destroying the game as it doesn't really fix the main issue. In the meantime, it is also destroying all the small and even medium players investments for which you clearly don't have any respect for. The game is sadly destined for bankruptcy.
> 
> The only thing that can save this game is hiring or consult with someone that runs a successful P2E game and knows how such a market works. I am not that expert, but am sure I can already recognize what won't work.

Nobody cares if you are leaving or not, Just go and don't drag us down who wants to stay. If you want to leave then leave. We are here to stay and we are not coming with you. We are staying! its our choice., Don't make negativity affects other people's decision. Just go!

**Comment by Vilandarion** [comment]
scam
